### PR TITLE
Display Languages feature

### PR DESCRIPTION
**Pull request by gogulram**

### CODE REVIEW

File: backup/NewUserRegistration.jsp [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE>
+<html lang="en" >
```
Comment (csys-fresher-batch-2021) [inline backup/NewUserRegistration.jsp]: Remove Registration page.

File: src/main/java/in/gokul/services/LanguageServices.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+
+
+public class LanguageServices {
+	private LanguageServices()
+	{
+		
+	}
+
+	private static final List <Languages> LanguageList= new ArrayList<>();
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/gokul/services/LanguageServices.java]: Follow naming convention

File: src/main/java/in/gokul/services/LanguageServices.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+          for (Languages languages : LanguageList) {
+				
+            if(language.equals(languages.getLanguage()))
+            {
+            	isAvailable=true;
+            }
+			}
+			
+		}
+
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/gokul/services/LanguageServices.java:23]: Format the code.

File: src/test/Login.jsp [hunk 4]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE>
+<html lang="en" >
```
Comment (csys-fresher-batch-2021) [inline src/test/Login.jsp]: Delete Login page

**Review by ghost — CHANGES_REQUESTED** [review]
Remove unnecessary files like Register and Login

File: backup/NewUserRegistration.jsp (hunk 1 above)
Comment (gogulram) [reply backup/NewUserRegistration.jsp]: removed

File: src/main/java/in/gokul/services/LanguageServices.java (hunk 2 above)
Comment (gogulram) [reply src/main/java/in/gokul/services/LanguageServices.java]: Changed

File: src/main/java/in/gokul/services/LanguageServices.java (hunk 3 above)
Comment (gogulram) [reply src/main/java/in/gokul/services/LanguageServices.java:23]: formated the code

File: src/test/Login.jsp (hunk 4 above)
Comment (gogulram) [reply src/test/Login.jsp]: deleted

File: src/main/java/in/gokul/model/Languages.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package in.gokul.model;
+
+public class Languages {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/gokul/model/Languages.java]: Rename to Language.

File: src/main/java/in/gokul/services/LanguageServices.java [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+	}
+
+	/**
+	 * accepts language as input checks for language available or not
+	 * 
+	 * @param language
+	 * @return
+	 */
+	public static boolean isLanguageAvailable(String language) {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/gokul/services/LanguageServices.java]: This method is not related to Display language feature.

**Review by ghost — CHANGES_REQUESTED** [review]
Review comments added

File: src/main/java/in/gokul/services/LanguageServices.java [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+	public static int totalLanguagesCount() {
+
+		return languageList.size();
+
+	}
+
+	public static List<Language> getAllLanguages() {
+		return languageList;
+
+	}
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/gokul/services/LanguageServices.java:40]: Format the code

File: src/main/webapp/languages.jsp [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+	padding: 70px 0;
+}
+</style>
+</head>
+<body>
+
+	<jsp:include page="header.jsp"></jsp:include>
+	<main class="container-fluid">
+		<%
+		String loggedInUsername = (String)session.getAttribute("Logged_In_User");
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/languages.jsp]: Remove LoggedInUserDetails

File: src/test/java/in/gokul/ServicesTest/LanguageTest.java [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+	}
+
+	
+
+	/**
+	 * gets the total languages Count
+	 */
+	@Test
+	public void TotalLanguagesCount() {
```
Comment (csys-fresher-batch-2021) [inline src/test/java/in/gokul/ServicesTest/LanguageTest.java]: Follow naming convention

File: src/main/java/in/gokul/validation/Validation.java [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+	 * 
+	 * @param language
+	 * @return
+	 */
+	public static boolean stringValidator(String language) {
+		boolean isvalid = true;
+		
+		try {
+
+			if ((language == null) || (language.trim().equals(" ")) ) {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/gokul/validation/Validation.java]: .equals("")

**Review by ghost — CHANGES_REQUESTED** [review]
Remove code which is not relevant to this feature.

File: src/main/java/in/gokul/services/LanguageServices.java (hunk 3 above)
Comment (csys-fresher-batch-2021) [reply src/main/java/in/gokul/services/LanguageServices.java:23]: thank you.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**